TaskManagement/src/TaskManagment/DB methods/Queries.java

```diff
@@ -0,0 +1,8 @@
+package TaskManagerProject;
+
+public interface Queries {
+   String CREATE_TASK_SQL_QUERY = "INSERT INTO task_management_system.task (task_name, description, tag, " +
+            "priority_level, due_date, assigned_user_id, created_by_user_id, category, stutes)" +
+            "VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?)";
+
```

Comment:
@HadeelSawalha Have you heared about `Hibernate` ORM for Java?
It is a widely used ORM framework (Object-Relational Mapper) that simplifies database access and management in Java applications.

I would suggest using it instead of writing string queries like that which is more error-prone and risky